scripts/docker-compose/install.sh

```diff
@@ -76,6 +76,32 @@ if [[ -z $DOMAIN_NAME ]]; then
 fi
 info "Using domain name: $DOMAIN_NAME 🌐"
 
+read -p "Is the domain on a public DNS? (y/n) " yn
+case $yn in 
+	y ) echo "$DOMAIN_NAME is on a public DNS";
+        ;;
+	n ) echo "$DOMAIN_NAME is on a private DNS";
+		# Maybe rename the docker-compose ?
+		# And add build image to the new docker-compose
+
+    # The docker-compose.private.yaml is the same as the docker-compose.yaml but with the chalice image changed to a local build image
+    # It's because we disable the verify in health.py
+
+		mv docker-compose.yaml docker-compose.public.yaml
+		mv docker-compose.private.yaml docker-compose.yaml
+
+    mv Caddyfile Caddyfile.public
+		mv Caddyfile.private Caddyfile
+		# this will change the calice image to a local build image
+		# Or change verify=False in health
+		sed -i 's/verify=True/verify=False/' ../../api/chalicelib/core/health.py
```

Comment:
we're not rebuilding the images with changed code. So I'd advise using the env variable in chalice.env